.github/workflows/build.yml

```diff
@@ -1,19 +1,29 @@
-name: Test & Build
-
+name: build
 on:
+  release:
+    types: [created]
```

Comment:
Will this be triggered when we "draft a new release"? I have used the type `published` in the past.

Reply:
Looks like it doesn't apply to drafts, although the docs are not very clear as to what's the difference: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
If we trust this StackOverflow response, using release scripts could only trigger a `publish` event: https://stackoverflow.com/a/61066906/775259

Not really using that at the moment, so happy to merge it if you want to keep all CI scripts the same, but worth knowing for the future.

Reply:
Pretty convinced it won't run for drafts but agree it may not run when a draft is published either, would need to test as the docs are unclear. I have a BAU ticket open to investigate this and will go back and update workflows based on that investigation.